node/resolvers/items.ts

```diff
@@ -214,6 +217,17 @@ export const mutations = {
 
     const cleanItems = orderItems.map(({ id, ...rest }) => rest)
 
+    // Validating subscriptions
+    let subscriptions
+    if (orderItems.length === 1 && orderItems[0].index !== undefined) {
```

Comment:
What if there's more than one item in the cart? Would it be always the first one?

Reply:
Thanks for your review

The solution is planned for the `updateItems()` mutation from [minicart](https://github.com/vtex-apps/minicart). The quantity of items in this case will be always one (already checked in code)  because the event is fired by a stepper within the context of a single item
![image](https://github.com/vtex-apps/checkout-graphql/assets/75274040/2fd4eec5-bcfc-4fb1-80d4-4d92f8b79e72)

Reply:
thnks for the explanation! lgtm ✨